app/controllers/discourse_assign/assign_controller.rb

```diff
@@ -122,6 +122,28 @@ def assigned
       render json: { topics: serialize_data(topics, AssignedTopicSerializer) }
     end
 
+    def group_members
+      limit = (params[:limit] || 50).to_i
+      offset = params[:offset].to_i
+
+      raise Discourse::InvalidParameters.new(:limit) if limit < 0 || limit > 1000
+      raise Discourse::InvalidParameters.new(:offset) if offset < 0
+      raise Discourse::NotFound.new if !params[:group_name].present?
+
+      group = Group.find_by(name: params[:group_name])
+      members = User
+        .joins("LEFT OUTER JOIN group_users g on users.id=g.user_id LEFT OUTER JOIN user_options uo on uo.user_id=users.id LEFT OUTER JOIN topic_custom_fields tcf ON tcf.value::int = users.id")
```

Comment:
Very nice! This should be nice and efficient for assignment counts with just one query ❤️